RELEASES.md

```diff
@@ -0,0 +1,77 @@
+# Grafana Releases
+
+## Release schedule
+
+Release cadence of first beta-release being cut is 6 weeks.
```

Comment:
Unclear what "being cut" means.

Also, what happens during those six weeks? @daniellee  told me there was a five-week cadence with one week rest. Is the first week usually planning?

You don't have to write yourself into a strict schedule, just give our imaginary new employee reader an idea of what that six-week release is like so they know what to expect.